### PR TITLE
Updated telegram group link

### PR DESCRIPTION
**Pull request by ailtonvivaz**
- [ ] This change has been discussed with the author.
- [ ] It implements / fixes issue #.

---

**Description:**
Updated telegram group link

### CODE REVIEW

**Comment by alejx** [comment]
Thanks!

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 <p align=center>
 <a href="https://iina.io">Website</a> ·
 <a href="https://github.com/iina/iina/releases">Releases</a> ·
-<a href="https://t.me/joinchat/AAAAAEBemW7dU8X7IHShwQ">Telegram Group</a>
+<a href="https://t.me/joinchat/IINAUsers">Telegram Group</a>
```
Comment (alexander-ii) [inline README.md]: The link should be https://t.me/IINAUsers

Comment (alejx) [reply README.md]: Thanks! Fixed.

**Comment by saagarjha** [comment]
TIL we apparently have a Telegram group?

**Comment by alejx** [comment]
Yes...

**Pull request merged**